### PR TITLE
Add Browser Reloading

### PR DESCRIPTION
**Pull request by ghickman**
Enable automatic reloading of the browser with [django-browser-reload](https://github.com/adamchainz/django-browser-reload) 🎉

### CODE REVIEW

**Review by iaindillingham — APPROVED** [review]

File: jobserver/urls.py [hunk 1]
```diff
@@ -272,6 +272,7 @@
     path("status/", Status.as_view(), name="status"),
     path("workspaces/", RedirectView.as_view(url="/")),
     path("__debug__/", include(debug_toolbar.urls)),
+    path("__reload__/", include("django_browser_reload.urls")),
```
Comment (inglesp) [inline jobserver/urls.py:275]: :seagull:

Given that the new library is only listed in the dev requirements, how does this work in production?

Comment (ghickman) [reply jobserver/urls.py:275]: [It doesn't](https://github.com/opensafely-core/job-server/runs/4584301749?check_suite_focus=true) 😬

I'll stick it in an if block first thing tomorrow.

Comment (inglesp) [reply jobserver/urls.py:275]: While we're here -- what about DDT?

Comment (ghickman) [reply jobserver/urls.py:275]: I was just thinking about that this morning, I'll pop browser reload into prod requirements, it saves possible settings.py problems down the line.  They're both turned off without DEBUG on.

**Pull request merged**